modules/oxide/src/less/theme/components/sidebar/sidebar-container.less

```
@@ -18,9 +18,20 @@
 
 .tox {
   .tox-sidebar-wrap {
-    display: flex;
-    flex-direction: row;
+    --tox-private-min-editing-area-width: 280px;
     flex-grow: 1;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)